src/docbkx/chapter13-getting-started-monitor.xml

```diff
@@ -30,14 +30,65 @@
 		</para>
 		<mediaobject>
 			<imageobject>
-				<imagedata fileref="monitor_01_login.jpg" format="JPG"
-					scalefit="1" />
+				<imagedata fileref="monitor_01_login.jpg" format="JPG" scalefit="1" />
 			</imageobject>
 		</mediaobject>
 		<para>In the community edition of DataCleaner, you will find
 			suggestions for login credentials directly on the screen, to get you
 			started quickly.
 		</para>
+		<section>
+			<title>Creating a resources folder on Windows</title>
+			<para>
+				DataCleaner monitor uses a special resources folder. Typically it is created automatically during 
+				installation as <emphasis>C:\Windows\Users\USER_NAME\.datacleaner</emphasis>. However, when using 
+				Apache Tomcat and its system service then this resources folder needs to be created manually as 
+				<emphasis>C:\Windows\System32\config\systemprofile\.datacleaner</emphasis>. 
+			</para>
+		</section>
+		<section>
+			<title>Example with a missing systemprofile resources folder</title>
+			<para>
+				Open your browser at <link xl:href="http://localhost:8080/DataCleaner-monitor/scheduling">
+				http://localhost:8080/DataCleaner-monitor/scheduling</link>. 
+				<mediaobject>
+					<imageobject>
+						<imagedata fileref="monitor_01_jobs.png" format="PNG" scalefit="1" />
+					</imageobject>
+				</mediaobject>
+				When you try to run, let's say, 'Customer duplicates' job in DataCleaner monitor, you get following 
+				exception (missing valid license). 
```

Comment:
What do you mean by that "default is dashboard"? I can view all the pages without license and even run "simple" jobs (probably those that do not require remote services).